scripts/buildPages.sh

```diff
@@ -9,5 +9,4 @@ done
 echo "All docs built."
 cd ./pages
 npx vite build
-sed -i 's/<link/<link data-polar="true"/g' ../pages_output/index.html
```

Comment:
I've re-added the deleted line since the mechanism is deprecated, but supposed to be kept in active testing and should keep working. I see changing this as an effort of the next major release. Maybe before that, if all customers adapted the new mechanism.